ci/licenses.sh

```diff
@@ -122,7 +122,7 @@ function verify_licenses() (
 
   local actualLicenseCount
   actualLicenseCount="$(tail -n 1 flutter/ci/licenses_golden/licenses_flutter | tr -dc '0-9')"
-  local expectedLicenseCount=2 # When changing this number: Update the error message below as well describing all expected license types.
```

Comment:
we got a lot of more licenses from forked chromium code. This is ok I think?

Reply:
cc @Hixie

Reply:
This has an impact on binary size for all customers.

If possible, it may be worth seeing if we can get the Chromium code to just use a single license, ideally one that we are already using/shipping, to avoid needing to do gymnastics around this and ship a larger license file.

Reply:
Presumably, this would also be resolved if we could just import the new code into `//third_party` instead of `//flutter/third_party`

Reply:
(Since the `//third_party` code would presumably all have one single license)

Reply:
I am surprised the third_party only use one license, are we presumably relicensed all third party code? last time I heard from @Hixie is we should not touch any licenses that are from other libraries.

Reply:
We can't change licenses for other libraries, but other libraries may choose to release with a new or unified license. We'd have to work with the upstream on this.